README.rst

```diff
@@ -104,11 +104,44 @@ on the command line:
 | | ``--json``      | Deep tokenization. Output token objects in JSON   |
 |                   | format, one per line.                             |
 +-------------------+---------------------------------------------------+
-| | ``--normalize`` | Normalize punctuation, causing e.g. quotes to be  |
-|                   | output in Icelandic form and hyphens to be        |
-|                   | regularized. This option is only applicable to    |
-|                   | shallow tokenization.                             |
-+-------------------+---------------------------------------------------+
+
+Other options can be specified on the command line:
+
++-----------------------------------+---------------------------------------------------+
+| | ``-n``                          | Normalize punctuation, causing e.g. quotes to be  |
+| | ``--normalize``                 | output in Icelandic form and hyphens to be        |
+|                                   | regularized. This option is only applicable to    |
+|                                   | shallow tokenization.                             |
++-----------------------------------+---------------------------------------------------+
+| | ``-s``                          | Input contains strictly one sentence per line.    |
+| | ``--one_sent_per_line``         |                                                   |
++-----------------------------------+---------------------------------------------------+
+| | ``-m``                          | Degree signal in tokens denoting temperature      |
+| | ``--convert_measurements``      | normalized (200° C -> 200 °C)                     |
++-----------------------------------+---------------------------------------------------+
+| | ``-a``                          | Additional annotation, usually handled by         |
+| | ``--with_annotation``           | GreynirPackage, added to tokens.                  |
++-----------------------------------+---------------------------------------------------+
+| | ``-p``                          | Numbers combined into one token with the          |
+| | ``--coalesce_percent``          | following token denoting percentage word forms    |
+|                                   | (prósent, prósentustig, hundraðshlutar)           |
++-----------------------------------+---------------------------------------------------+
+| | ``-g``                          | Composite glyphs replaced with a single           |
+| | ``--replace_composite_glyphs``  | code point, such as a ́' -> á                     |
```

Comment:
Change this documentation to keep_composite_glyphs (and modify explanation accordingly)